src/autopick/autopick-describer.c

```diff
@@ -349,28 +353,39 @@ void describe_autopick_en(char *buff, autopick_type *entry, autopick_describer *
 	if (IS_FLG(FLG_BOOSTED))
 	{
 		describer->body_str = "weapons";
-		whose_str[whose_n++] = "damage dice is bigger than normal";
+		whose_str[whose_n] = "damage dice is bigger than normal";
+		whose_arg_str[whose_n] = "";
+		++whose_n;
 	}
 
 	if (IS_FLG(FLG_MORE_DICE))
 	{
-		static char more_than_desc_str[] =
-			"maximum damage from dice is bigger than __";
 		describer->body_str = "weapons";
-
-		sprintf(more_than_desc_str + sizeof(more_than_desc_str) - 3,
-			"%d", entry->dice);
-		whose_str[whose_n++] = more_than_desc_str;
+		whose_str[whose_n] =
+			"maximum damage from dice is bigger than ";
+		if (arg_n < (int) (sizeof(arg_str) / sizeof(arg_str[0]))) {
+			sprintf(arg_str[arg_n], "%d", entry->dice);
```

Comment:
As I commented in #33, we want to use snprintf in future added code, so please rewrite sprintf with snprintf.

Reply:
The second commit converts the sprintf() calls to snprintf() for autopick-describer.c.